docs/content/3.guide/1.writing/4.json.md

```diff
@@ -0,0 +1,38 @@
+---
+title: JSON
+description: 'Nuxt Content can query JSON formatted content.'
+---
+
+# JSON
+
+Nuxt Content can query JSON formatted content. The output contains all the document properties at its root.
```

Comment:
It is worth mentioning that output goes under `body` key if content contains array data

Reply:
Ok, very clear now, thanks!

Reply:
Thank you.  What am I missing?  Nothing is returned .. can't figure out what I'm doing wrong


`content/links.json` 
```json
[
  {
    "title": "Link title",
    "url": "https://localhost/",
    "tags": [
      "info",
      "blah"
    ]
  },
  {
    "title": "Another link",
    "url": "https://localhost/",
    "tags": [
      "info",
      "things"
    ]
  }
]
```


```js
async asyncData ({ $content }) {
  const books = await $content('books').sortBy('createdAt', 'desc').fetch()
  const links = await $content('links').fetch()
  return { books,links }
}
```

![image](https://user-images.githubusercontent.com/89231731/170807859-83dc9642-fdcc-4c79-aeac-0d3196ff427c.png)